USE_CASES.md

```diff
@@ -2,6 +2,7 @@ This documentation provides examples for specific use cases. Please [open an iss
 
 # Table of Contents
 
+* [Dynamic Template Data](#dynamic-template-data)
```

Comment:
In our UI, we are calling these Transactional Templates and the previous Transactional Templates are now called Legacy Templates.